### PR TITLE
chore: revert batching algorithm

### PR DESCRIPTION
**Pull request by shimoncohen**
| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✖                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✔                                                                       |

### CODE REVIEW

File: src/createPackage/models/createPackageManager.ts [hunk 1]
```diff
@@ -271,15 +282,20 @@ export class CreatePackageManager {
...
+    // TODO: remove and replace with `generateTileGroups` that is commented, when multiple tasks for GPKG target is possible
     const batches: ITileRange[] = [];
     featuresRecords.forEach((record) => {
-      const recordBatches = this.generateTileGroups(
-        record.geometry as Polygon | MultiPolygon,
-        layerMetadata.footprint as Polygon | MultiPolygon,
-        record.zoomLevel
-      );
-      batches.push(...recordBatches);
+      const recordBatches = bboxToTileRange(record.sanitizedBox as BBox2d, record.zoomLevel);
```
Comment (asafMasa) [inline src/createPackage/models/createPackageManager.ts]: fix all zoom levels for each feature

**Review by asafMasa — CHANGES_REQUESTED** [review]

File: src/createPackage/models/createPackageManager.ts (hunk 1 above)
Comment (shimoncohen) [reply src/createPackage/models/createPackageManager.ts]: Fixed.

File: src/createPackage/models/createPackageManager.ts [hunk 2]
```diff
@@ -271,15 +283,23 @@ export class CreatePackageManager {
...
     featuresRecords.forEach((record) => {
-      const recordBatches = this.generateTileGroups(
-        record.geometry as Polygon | MultiPolygon,
-        layerMetadata.footprint as Polygon | MultiPolygon,
-        record.zoomLevel
-      );
-      batches.push(...recordBatches);
+      for (let zoom = 0; zoom <= record.zoomLevel; zoom++) {
+        const recordBatches = bboxToTileRange(record.sanitizedBox as BBox2d, zoom);
+        console.log(recordBatches);
```
Comment (ronenkapelian) [inline src/createPackage/models/createPackageManager.ts]: remove console.log

File: tests/integration/createPackage/createExportPackage.spec.ts [hunk 3]
```diff
@@ -58,17 +58,18 @@ describe('Export by ROI', function () {
         crs: 'EPSG:4326',
         priority: 0,
       };
+
       findLayerSpy.mockResolvedValue(layerFromCatalog);
       checkForExportDuplicateSpy.mockResolvedValue(undefined);
       validateFreeSpaceSpy.mockResolvedValue(true);
-      generateTileGroupsSpy.mockReturnValue([]);
+      // generateTileGroupsSpy.mockReturnValue([]);
```
Comment (ronenkapelian) [inline tests/integration/createPackage/createExportPackage.spec.ts:66]: add comment of "todo-after bug fix it will be returned..."etc...

**Review by ronenkapelian — CHANGES_REQUESTED** [review]
run local + local docker on azure and validate creation of valid gpkg in getMap API and ROI api

**Review by asafMasa — CHANGES_REQUESTED** [review]
👍

**Review by ronenkapelian — APPROVED** [review]

**Review by asafMasa — APPROVED** [review]

**Pull request merged**